.github/workflows/build_msi.yml

```diff
@@ -18,6 +18,10 @@ on:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
@hugovk, shouldn't those match the build.yml group name, too?

Reply:
The `github.workflow` bit should deal with that:

> To only cancel in-progress runs of the same workflow, you can use the `github.workflow` property to build the concurrency group

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency